saleor/webhook/transport/synchronous/circuit_breaker/breaker_board.py

```diff
@@ -21,16 +28,57 @@ class BreakerBoard:
     webhook execution.
     """
 
-    def __init__(self, storage: Storage, failure_threshold: int):
-        self.storage = storage
+    def configure(
+        self,
+        storage: Storage,
+        mode: str,
+        failure_threshold: float,
+        cooldown_seconds: int,
+        ttl: int,
+    ):
+        self.failure_threshold = failure_threshold or settings.BREAKER_FAILURE_THRESHOLD
+        self.cooldown_seconds = cooldown_seconds or settings.BREAKER_COOLDOWN
+        self.ttl = ttl or settings.BREAKER_TTL
+        if mode == BreakerMode.FIXED and self.failure_threshold < 1:
+            logger.info("Circuit breaker threshold invalid.")
+            self.mode = BreakerMode.NONE
+            return
+        elif mode == BreakerMode.PERCENTAGE and (
+            self.failure_threshold < 1 or self.failure_threshold > 99
+        ):
+            logger.info("Circuit breaker threshold invalid.")
+            self.mode = BreakerMode.NONE
+            return
+        self.storage = storage or InMemoryStorage()
+        self.mode = mode
+        if self.cooldown_seconds < 1 or self.failure_threshold < 1 or self.ttl < 1:
+            logger.info("Circuit breaker configuration invalid.")
+            self.mode = BreakerMode.NONE
+
+    def __init__(
+        self,
+        storage: Optional[Storage] = None,
+        mode: Optional[str] = None,
+        failure_threshold: Optional[float] = None,
+        cooldown_seconds: Optional[int] = None,
+        ttl: Optional[int] = None,
+    ):
+        self.mode = mode or settings.BREAKER_MODE
+        if (
+            not self.mode
+            or self.mode == BreakerMode.NONE
+            or self.mode not in BreakerMode.OPTIONS
+        ):
+            self.mode = BreakerMode.NONE
+            logger.info("Circuit breaker feature is disabled.")
```

Comment:
If configuration options (`MODE`) are not valid - raise an error, do not let misconfigured Saleor start.

I think we'd rather see Saleor fail right away than find out later that breaker board was misconfigured.